solidity/contracts/bridge/BridgeGovernance.sol

```diff
@@ -0,0 +1,1542 @@
+
+    event RedemptionTimeoutSlashingAmountUpdateStarted(
+        uint96 newRedemptionTimeoutSlashingAmount,
+        uint256 timestamp
+    );
+    event RedemptionTimeoutSlashingAmountUpdated(
+        uint96 redemptionTimeoutSlashingAmount
+    );
+
+    event RedemptionTimeoutNotifierRewardMultiplierUpdateStarted(
+        uint64 newRedemptionTimeoutNotifierRewardMultiplier,
+        uint256 timestamp
+    );
+    event RedemptionTimeoutNotifierRewardMultiplierUpdated(
+        uint64 redemptionTimeoutNotifierRewardMultiplier
+    );
+
+    event MovingFundsTxMaxTotalFeeUpdateStarted(
+        uint64 newMovingFundsTxMaxTotalFee,
+        uint256 timestamp
+    );
+    event MovingFundsTxMaxTotalFeeUpdated(uint64 movingFundsTxMaxTotalFee);
+
+    event MovingFundsDustThresholdUpdateStarted(
+        uint64 newMovingFundsDustThreshold,
+        uint256 timestamp
+    );
+    event MovingFundsDustThresholdUpdated(uint64 movingFundsDustThreshold);
+
+    event MovingFundsTimeoutResetDelayUpdateStarted(
+        uint32 newMovingFundsTimeoutResetDelay,
+        uint256 timestamp
+    );
+    event MovingFundsTimeoutResetDelayUpdated(
+        uint64 movingFundsTimeoutResetDelay
+    );
+
+    event MovingFundsTimeoutUpdateStarted(
+        uint64 newMovingFundsTimeout,
+        uint256 timestamp
+    );
+    event MovingFundsTimeoutUpdated(uint64 movingFundsTimeout);
+
+    event MovingFundsTimeoutSlashingAmountUpdateStarted(
+        uint96 newMovingFundsTimeoutSlashingAmount,
+        uint256 timestamp
+    );
+    event MovingFundsTimeoutSlashingAmountUpdated(
+        uint96 movingFundsTimeoutSlashingAmount
+    );
+
+    event MovingFundsTimeoutNotifierRewardMultiplierUpdateStarted(
+        uint256 newMovingFundsTimeoutNotifierRewardMultiplier,
+        uint256 timestamp
+    );
+    event MovingFundsTimeoutNotifierRewardMultiplierUpdated(
+        uint256 movingFundsTimeoutNotifierRewardMultiplier
+    );
+
+    event MovedFundsSweepTxMaxTotalFeeUpdateStarted(
+        uint64 newMovedFundsSweepTxMaxTotalFee,
+        uint256 timestamp
+    );
+    event MovedFundsSweepTxMaxTotalFeeUpdated(
+        uint64 movedFundsSweepTxMaxTotalFee
+    );
+
+    event MovedFundsSweepTimeoutUpdateStarted(
+        uint32 newMovedFundsSweepTimeout,
+        uint256 timestamp
+    );
+    event MovedFundsSweepTimeoutUpdated(uint32 movedFundsSweepTimeout);
+
+    event MovedFundsSweepTimeoutSlashingAmountUpdateStarted(
+        uint96 newMovedFundsSweepTimeoutSlashingAmount,
+        uint256 timestamp
+    );
+    event MovedFundsSweepTimeoutSlashingAmountUpdated(
+        uint96 movedFundsSweepTimeoutSlashingAmount
+    );
+
+    event MovedFundsSweepTimeoutNotifierRewardMultiplierUpdateStarted(
+        uint256 newMovedFundsSweepTimeoutNotifierRewardMultiplier,
+        uint256 timestamp
+    );
+    event MovedFundsSweepTimeoutNotifierRewardMultiplierUpdated(
+        uint256 movedFundsSweepTimeoutNotifierRewardMultiplier
+    );
```

Comment:
The type should be `uint32`:
```
uint32 movedFundsSweepTimeoutNotifierRewardMultiplier;
```
Let's change it everywhere this parameter occurs (events, functions).

Reply:
Done in [1e40d72](https://github.com/keep-network/tbtc-v2/pull/329/commits/1e40d7285883be2f48ca69bfc103ca11e0d7bc12)